web/src/main/webapp/WEB-INF/config-security/openid-configuration-jose.json

```diff
@@ -0,0 +1 @@
+{"token_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/oauth2/v2.0/token","token_endpoint_auth_methods_supported":["client_secret_post","private_key_jwt","client_secret_basic"],"jwks_uri":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/discovery/v2.0/keys","response_modes_supported":["query","fragment","form_post"],"subject_types_supported":["pairwise"],"id_token_signing_alg_values_supported":["RS256"],"response_types_supported":["code","id_token","code id_token","id_token token"],"scopes_supported":["openid","profile","email","offline_access"],"issuer":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/v2.0","request_uri_parameter_supported":false,"userinfo_endpoint":"https://graph.microsoft.com/oidc/userinfo","authorization_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/oauth2/v2.0/authorize","device_authorization_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/oauth2/v2.0/devicecode","http_logout_supported":true,"frontchannel_logout_supported":true,"end_session_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/oauth2/v2.0/logout","claims_supported":["sub","iss","cloud_instance_name","cloud_instance_host_name","cloud_graph_host_name","msgraph_host","aud","exp","iat","auth_time","acr","nonce","preferred_username","name","tid","ver","at_hash","c_hash","email"],"kerberos_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/kerberos","tenant_region_scope":"EU","cloud_instance_name":"microsoftonline.com","cloud_graph_host_name":"graph.windows.net","msgraph_host":"graph.microsoft.com","rbac_url":"https://pas.windows.net"}
```

Comment:
Should this file really be added to the repo - it seems like it is a test file from local system?  I also don't see any reference to this file.

Reply:
yes - this was an accident.  will delete

Reply:
thanks for the review, @ianwallen